### PR TITLE
Add activestate OIDC docs

### PR DESCRIPTION
**Pull request by th3coop**

### CODE REVIEW

File: warehouse/templates/manage/account/publishing.html
Comment (woodruffw) [inline warehouse/templates/manage/account/publishing.html:28]: This was probably my fault originally 🙂 

```suggestion
    Read more about GitHub Actions' OpenID Connect support <a href="{{ href }}">here</a>.
```

**Comment by woodruffw** [comment]
Thanks @th3coop! I did a quick scan; I'll also stand this up locally in a bit to confirm the render is as expected.

File: warehouse/templates/manage/account/publishing.html
Comment (th3coop) [reply warehouse/templates/manage/account/publishing.html:28]: Naw, I was mucking around, copy and pasting stuff at the end of the day yesterday and was going to come back to this mistake.  Thanks for the change suggestion!

File: docs/user/trusted-publishers/adding-a-publisher.md
Comment (di) [inline docs/user/trusted-publishers/adding-a-publisher.md:100]:
```suggestion
    For ActiveState, you need to provide the name of the ActiveState project,
```
I think this makes it a bit more clear which project name this is.

File: docs/user/trusted-publishers/adding-a-publisher.md
Comment (di) [inline docs/user/trusted-publishers/adding-a-publisher.md:101]:
```suggestion
    the ActiveState organization that project belongs to, and the ActiveState user performing
```

File: docs/user/trusted-publishers/creating-a-project-through-oidc.md
Comment (di) [inline docs/user/trusted-publishers/creating-a-project-through-oidc.md:66]:
```suggestion
    updating a project. You need to provide the name of the ActiveState project,
```

File: docs/user/trusted-publishers/creating-a-project-through-oidc.md
Comment (di) [inline docs/user/trusted-publishers/creating-a-project-through-oidc.md:67]:
```suggestion
    the ActiveState organization that project belongs to, and the ActiveState user who will be performing
```

File: docs/user/trusted-publishers/security-model.md
Comment (di) [inline docs/user/trusted-publishers/security-model.md:161]:
```suggestion
    an environment variable along with your release artifact that will be uploaded. The
```

File: docs/user/trusted-publishers/adding-a-publisher.md
Comment (di) [inline docs/user/trusted-publishers/adding-a-publisher.md:102]: I'm assuming this URL will start resolving sometime after we enable this feature?

File: docs/user/trusted-publishers/security-model.md
Comment (th3coop) [reply docs/user/trusted-publishers/security-model.md:172]: Correct.  They will be up soon.

File: docs/user/trusted-publishers/using-a-publisher.md
Comment (di) [inline docs/user/trusted-publishers/using-a-publisher.md:273]: Is there something we can link to that details the supported platforms/architectures/etc.?

Comment (rawktron) [reply docs/user/trusted-publishers/using-a-publisher.md:273]: Good call. It's available via the API, but we'll look at putting something together in the linked docs that's easy to reference.

File: docs/user/trusted-publishers/adding-a-publisher.md
Comment (th3coop) [reply docs/user/trusted-publishers/adding-a-publisher.md:100]: I agreed!

File: docs/user/trusted-publishers/adding-a-publisher.md
Comment (rawktron) [reply docs/user/trusted-publishers/adding-a-publisher.md:102]: Yes, correct. It should be live soon. We'll get it set up so that it can be viewable before we go live.

**Review by woodruffw — APPROVED** [review]
LGTM! I think we can merge this into the bigger WIP docs branch as-is, even if there are still 404s.

(Looks like the CI needs to be kicked over, but I don't have the permission for that.)

**Comment by th3coop** [comment]
@woodruffw  we have a couple more changes coming.  Pete will update.  I'd like to squash those `Review Edit` commits out of the history as well when Pete is done.

**Comment by di** [comment]
(No need to squash out commits, they'll get squashed when we merge!)

**Comment by th3coop** [comment]
OH right, I knew that!  Thanks for the reminder @di.

**Comment by th3coop** [comment]
@di or @woodruffw, is it possible to see the logs of test.pypi.org?  We're having trouble tracking down an issue where test.pypi.org rejects an OIDC token that is accepted by a locally runnning warehouse with the same/similar pending publisher configured.

What's the best way to proceed to debug this?

**Comment by rawktron** [comment]
> @woodruffw we have a couple more changes coming. Pete will update. I'd like to squash those `Review Edit` commits out of the history as well when Pete is done.

Ok -- I pushed the updates I needed to push for the docs. @woodruffw @di @th3coop

**Comment by th3coop** [comment]
> @di or @woodruffw, is it possible to see the logs of test.pypi.org? We're having trouble tracking down an issue where test.pypi.org rejects an OIDC token that is accepted by a locally runnning warehouse with the same/similar pending publisher configured.
> 
> What's the best way to proceed to debug this?

Sorry for the noies @di and @woodruffw!  I'm pretty sure we figured it out.

**Comment by di** [comment]
Great @th3coop @rawktron -- is this good to go on your end then?

**Comment by rawktron** [comment]
> Great @th3coop @rawktron -- is this good to go on your end then?

@di So on the docs side, *fingers crossed* yes this should be complete. On the technical side, we're still working through some integration issues to get everything working end to end with test.pypi.org, but that shouldn't (famous last words) alter the instructions here. We also still need to flip those few external pages live as well to get rid of the 404s.

**Comment by di** [comment]
OK, I'll hold off on merging until you confirm you've got everything right technically -- I'm fine with merging this with 404s for now since it won't go live until we merge https://github.com/pypi/warehouse/pull/15192.

**Comment by rawktron** [comment]
> OK, I'll hold off on merging until you confirm you've got everything right technically -- I'm fine with merging this with 404s for now since it won't go live until we merge #15192.

Alright, everything seems to be working end-to-end technically now, and I pushed one more update to the docs to reflect a few tweaks. So I think we're good to merge into 15192. Thanks for your patience!

**Pull request merged**
